util/backup/advanced/backup-automated.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
```

Comment:
assuming these scripts were taken from https://github.com/uzh-bf/klicker-uzh/pull/4777 and modified heavily? so we would need to close and not merge the other branch to prevent a lot of conflicts